src/components/CustomDropdown/style.css

```diff
@@ -0,0 +1,48 @@
+.menu {
+    width: var(--dropdown-menu-width);
+    background-color: var(--dark-theme-dropdown-menu-bg);
+    border-radius: 3px;
+    padding: 8px;
+}
+
+.menu.narrow {
+    width: calc(var(--dropdown-menu-width) - 50px);
```

Comment:
Nit: This seems a little magic number-y, would it make more sense to either define a narrow width variable or make the width set by `fit-content` or something?

Reply:
Much better, thank you